model/AbstractRdsResultStorage.php

```diff
@@ -727,4 +731,13 @@ public function createTableConstraints(Table $variablesTable, Table $resultsTabl
             self::VARIABLES_FK_NAME
         );
     }
+
+    /**
+     * @param array $data
+     * @return array
+     */
+    protected function getTypes(array $data = [])
```

Comment:
There is smthing wrong with this method You are not using data and you always return an empty array?

Reply:
what if we move this 
```
    /**
     * @inheritDoc
     */
    protected function getTypes(array $data = [])
    {
        return [
            ParameterType::STRING,
            ParameterType::STRING,
            ParameterType::STRING,
            ParameterType::STRING,
            ParameterType::STRING,
            ParameterType::STRING,
            null,
            ParameterType::STRING,
        ];
    }
}
```
to the abstract ? will it hurt, 
and ofc for every other implementation we can overwrite